### PR TITLE
FIX: seed node could not connect to peers

### PR DESCRIPTION
**Pull request by mariusmihaic**
## Description of the reasoning behind the pull request (what feature was missing / how the problem was manifesting itself / what was the motive behind the refactoring)
- When running a local testnet, seednode could not connect to any peer
  
## Proposed Changes
- Generated new seednode p2pkey in `seednode/config` and replaced it in other config files.
- When calling `copySeednodeConfig`, we also copy seednode key

## Testing procedure
- Run a local testnet with standard procedure:
1. Go to `scripts/testnet`
2. `./clean.sh`
3. `./config.sh`
4. `./start.sh`. Here, seednode should connect to all peers
5. `./stop.sh` when finished

### CODE REVIEW

**Review by miiu96 — DISMISSED** [review]

File: cmd/seednode/config/p2pKey.pem [hunk 1]
```diff
@@ -0,0 +1,4 @@
+-----BEGIN PRIVATE KEY for 16Uiu2HAmKBCtAqnmawCJ5nxd9AccU3fj4sjA8WA3oksqeZgcsJJv-----
```
Comment (AdoAdoAdo) [inline cmd/seednode/config/p2pKey.pem:1]: can you generate the seed node pem file from the testnet scripts, into the relevant seednode folder?

**Review by AdoAdoAdo — CHANGES_REQUESTED** [review]

**Review by AdoAdoAdo — APPROVED** [review]

**Review by AdoAdoAdo — DISMISSED** [review]

**Review by bogdan-rosianu — DISMISSED** [review]

File: scripts/testnet/variables.sh [hunk 2]
```diff
@@ -87,7 +87,7 @@ export PORT_ORIGIN_VALIDATOR="21500"
 export PORT_ORIGIN_VALIDATOR_REST="9500"
 
 # Address of the Seednode. Will be written to the p2p.toml file of the Nodes
-export P2P_SEEDNODE_ADDRESS="/ip4/127.0.0.1/tcp/$PORT_SEEDNODE/p2p/16Uiu2HAkw5SNNtSvH1zJiQ6Gc3WoGNSxiyNueRKe6fuAuh57G3Bk"
+export P2P_SEEDNODE_ADDRESS="/ip4/127.0.0.1/tcp/$PORT_SEEDNODE/p2p/"
```
Comment (iulianpascalau) [inline scripts/testnet/variables.sh:90]: is this correct? It is missing the peer id (pid) part.

Please use a PEER_ID export var

**Review by iulianpascalau — CHANGES_REQUESTED** [review]

File: scripts/testnet/variables.sh (hunk 2 above)
Comment (miiu96) [reply scripts/testnet/variables.sh:93]: refactored

File: scripts/testnet/variables.sh [hunk 3]
```diff
@@ -79,6 +79,9 @@ export ALWAYS_UPDATE_CONFIGS=1
 # Always rebuild Arwen from its sources and copy the executable to the testnet folder.
 export ALWAYS_BUILD_ARWEN=1
 
+# IP of the seednode
+export SEEDNODE_IP="127.0.0.1"
```
Comment (iulianpascalau) [inline scripts/testnet/variables.sh:83]: do we still need L93?

Comment (miiu96) [reply scripts/testnet/variables.sh:83]: no, removed

**Review by iulianpascalau — APPROVED** [review]

**Review by AdoAdoAdo — APPROVED** [review]

**Pull request merged**